applicationLibrary/molecularDynamics/molecularDynamicsLibrary/ParticlePropertiesLibrary.h

```diff
@@ -245,6 +274,31 @@ class ParticlePropertiesLibrary {
    */
   static double calcShift6(double epsilon24, double sigmaSquared, double cutoffSquared);
 
+  /**
+   * Returns the precomputed mixed epsilon * 24.
+   * @param  i Id of site one.
+   * @param  j Id of site two.
+   * @param  k Id of site three.
+   * @return nu_ijk
+   */
+  inline floatType getMixingNu(intType i, intType j, intType k) const {
```

Comment:
But why? The functionality is the same as before no?
Can you show me how you see this? :D

Reply:
this `inline` is redundant because member functions that are implemented within a class are implicitly considered `inline` (see the second sentence of the [documentation](https://en.cppreference.com/w/cpp/language/inline).